cli/cmd/top.go

```diff
@@ -642,16 +670,18 @@ func (t *topTable) adjustColumnWidths() {
 				t.columns[i].width = cellWidth
 			}
 		}
+		tablewidth = tablewidth +  t.columns[i].width + columnGap
```

Comment:
I think your math is thrown off here because of the `continue` on line 663.  This does not take into account the width of non-flexible columns.  I suggest changing it so that instead of using continue, we just put the column resizing code inside the `if col.flexible` block.

Alternatively, we can revert the changes to this function and create a new dedicated function for calculating table width.  Either way, we should be able to get rid the `columnGap` constant.

Reply:
I have made changes considering all your inputs :)